packages/contracts-bedrock/test/setup/CommonTest.sol

```diff
@@ -68,21 +70,27 @@ contract CommonTest is Test, Setup, Events {
             deploy.cfg().setUseInterop(true);
         }
 
+        if (isForkTest) {
+            // Skip any test suite which uses a nonstandard configuration.
+            if (useAltDAOverride || useLegacyContracts || customGasToken != address(0) || useInteropOverride) {
+                vm.skip(true);
+            }
+
+            // Modifying these values on a fork test causes issues.
+            vm.warp(deploy.cfg().l2OutputOracleStartingTimestamp() + 1);
+            vm.roll(deploy.cfg().l2OutputOracleStartingBlockNumber() + 1);
+            vm.fee(1 gwei);
```

Comment:
What issues does it cause? Also not following what this means since we are in a block where `isForkTest` is true, so are we causing the issues by modifying these values?

Reply:
Yes, this diff fixed quite a few tests

Looking at the actual [deploy config values](https://github.com/ethereum-optimism/optimism/blob/upgrade-against-registry/packages/contracts-bedrock/deploy-config/hardhat.json#L17-L18) used in the test, this code block simplifies to: 

```
  vm.warp(2);
  vm.roll(2); 
  vm.fee(1 gwei);
```

This is quite disconnected from the actual state of the network, so we should expect that things break (ie. 2 step withdrawals and fault proving) when the global block number becomes significantly different from the block number which the contract state is based on.

Reply:
Oh, facepalm. The logic here was wrong. I think I had it working then broke it, and given so many failing tests, it's easy to not notice that.

```
if (isForkTest) {
  ....
} else {
    // Modifying these values on a fork test causes issues.
    vm.warp(deploy.cfg().l2OutputOracleStartingTimestamp() + 1);
    vm.roll(deploy.cfg().l2OutputOracleStartingBlockNumber() + 1);
    vm.fee(1 gwei);
}
```